packages/database/src/migrations/20200914045259-RemoveSomeLaosSchoolsSchoolIndicatorsEiEMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
This migration is for removing some existing overlays in School Indicators EiE groups:

To remove the following indicators from School Indicators EiE list:

> Dormitory schools
> Functioning toilets
> Students require psychosocial support
> Teachers using resources on MoES website
> Textbooks received since June 2020 (SchCVD004)
> Functioning water filters (SchCVD009)
> Internet connection available (SchFF002)
> Telephone available (SchCVD026a)